src/NServiceBus.Testing.Fakes/TestingLoggerFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+    using System.IO;
+    using Logging;
+
+    /// <summary>
+    /// Logger factory which allows to log to a text writer.
+    /// </summary>
+    public class TestingLoggerFactory : LoggingFactoryDefinition
+    {
+        /// <summary>
+        /// Creates a new instance of a testing logger factory.
+        /// </summary>
+        public TestingLoggerFactory()
+        {
+            level = new Lazy<LogLevel>(() => LogLevel.Debug);
+            writer = new Lazy<TextWriter>(() => new StringWriter());
```

Comment:
I've never been a fan of "the implementation of dispose does nothing so we can skip it". The whole point of the interface is so you don't have to worry about the internals (and the internals could change in the future anyway) - you just have to respect the contract.

That being said I'm ok with leaving this as is - it's a testing class not likely to become a long-lived production issue.

So :+1: for your explanation.